lib/apia/open_api/specification.rb

```diff
@@ -14,14 +14,14 @@ class Specification
 
       OPEN_API_VERSION = "3.0.0" # The Ruby client generator currently only supports v3.0.0 https://openapi-generator.tech/
 
-      def initialize(api, base_url, name, info = {}, external_docs = {})
+      def initialize(api, base_url, name, additions)
```

Comment:
do we want to make these `additions` required?

what if we did something like this (untested):
```ruby
def initialize(api, base_url, name, additions = {})
  default_additions = { info: {}, external_docs: {}, security_schemes: {} }
  additions = default_additions.merge(additions)
end
```
That would allow the caller to optionally supply `info` and nothing else if they wanted.